backend/lib/services/projects.js

```diff
@@ -148,22 +151,12 @@ exports.list = async function ({ user, qs = {} }) {
     return hasGroupMembership || hasUserMembership || hasServiceAccountMembership
   }
 
-  const phases = _
-    .chain(qs)
-    .get('phase', 'Ready')
-    .split(',')
-    .compact()
-    .value()
```

Comment:
> We need to discuss if we want to stop the user from working with unready projects

> I completely removed the phase filtering logic and qs param from the backend. I think we don't need it anymore, need to check

We can't be sure if the user can work with the project.

A project can be in failed state in case the namespace could not be claimed. In this case you can't work with the project https://github.com/gardener/gardener/blob/master/pkg/controllermanager/controller/project/project_control_reconcile.go#L63-L70. 
It can also be in a failed state when the RBAC resources could not be reconciled https://github.com/gardener/gardener/blob/master/pkg/controllermanager/controller/project/project_control_reconcile.go#L110-L114. E.g. you are assigned to the project as member with a specific role, but the rolebinding was not yet updated and thus you have no permissions or the other way around.

Reply:
As discussed, I changed it to exclude only projects that are in phase `Pending`. Projects may turn from `Pending` to `Failed`, e.g. as you described. However, we agreed that users should still see those projects. Gardener should provide appropriate error messages if the user tries to perform an action that includes creating resources in the namespace, like creating a shoot or secret.